### ISSUE TITLE
RAM Footprint difference between Telemetry Dashbord and App Metadata Analyzer

### ISSUE THREAD

**Issue by ftonial** [eapowertools/qs-telemetry-dashboard#48]
Hello, I would like to know why the RAM Footprint difference between Telemetry Dashbord and App Metadata Analyzer.

![image](https://user-images.githubusercontent.com/38221113/119844156-92cf2c00-bede-11eb-8680-96bf724411cf.png)

**Comment by levi-turner** [comment]
Ultimately it's because it's measuring RAM in two different ways. 

1. The QIX Performance logger is a global process which uses less isolation in polling requests for their memory use. In plain English, it should not be treated as a byte-perfect representation of memory usage by request. 
2. Comparing the `static_byte_size` element of the App Metadata endpoint (which is representative of the base data model) to the `AppOpen` event from the QIX Performance logs is a bit of stretch. When a user opens an app, they certainly need to open the base data model into RAM. But the Engine _also_ loads calculated dimensions and fields on the fly into memory ([reference](https://community.qlik.com/t5/Qlik-Design-Blog/Calculated-Fields/ba-p/1694383)). So Telemetry is more representative of the _actual_ in memory size of the application than the App Metadata endpoint.

My understanding from the Engine developers at Qlik is that the App Metadata endpoint is more reliable in some ways but not in others. It's more isolated in its scope (contra point 1 above). But contra point 2, it's very narrowly focused (to just `AppOpen` events and `DoReload` / `DoReloadEx` events) and can miss quite a bit.

**Comment by ftonial** [comment]
Thanks Levi, I understood you very well.
Best regards.

**Comment by ftonial** [comment]
Thanks Levi, I understood you very well.
Best regards.
